routers/api/v1/repo/issue.go

```diff
@@ -51,6 +51,11 @@ func SearchIssues(ctx *context.APIContext) {
 	//   description: repository to prioritize in the results
 	//   type: integer
 	//   format: int64
+	// - name: tp
```

Comment:
Is "tp" supposed to mean "type"? If so, suggesting rename for clarity.

Reply:
hmm you are right - for non gita familiar people it looks weared ..